<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDate</class>
 <widget class="QWidget" name="SearchDate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>53</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QDateTimeEdit" name="dateTimeEdit"/>
   </item>
   <item>
    <widget class="QLineEdit" name="format">
     <property name="text">
      <string>dd.MM.yyyy</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="timestamp">
     <property name="text">
      <string>Timestamp</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="date_str">
     <property name="text">
      <string>String</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
